app/assets/scripts/a11n/rules.js

```diff
@@ -53,6 +53,12 @@ export function defineRulesFor(user) {
         status: PUBLICATION
       });
       allow('access-comments', 'document-version');
+      allow('download-journal-pdf', 'document-version', {
+        'authors.sub': user.id
+      });
+      allow('download-journal-pdf', 'document-version', {
+        'reviewers.sub': user.id
+      });
```

Comment:
The curator does not belong to any document so these conditions will never be `true`.
If the curator needs to have access this is enough:
```
allow('download-journal-pdf', 'document-version');
```